packages/graphql-hooks/src/useClientRequest.js

```diff
@@ -72,7 +72,8 @@ function useClientRequest(query, initialOpts = {}) {
     )
   }
 
-  const client = React.useContext(ClientContext)
+  const contextClient = React.useContext(ClientContext)
+  const client = initialOpts.client || contextClient
```

Comment:
Minor:

```suggestion
  const client = initialOpts.client || React.useContext(ClientContext)
```

Reply:
You can't do this because then you have conditional hooks.

Reply:
Doh! Completely forgot about that...